### DISCUSSION TITLE
My application is pending

### DISCUSSION THREAD

**Discussion by UPUPUPX** [community/community#22718] [New to GitHub]
<p>My application was submitted on July 27, but it has not been approved until August 10.</p>

**Comment by wordsoup** [comment, marked as answer]
<p>Hi <a class="mention" href="/upupupx">@UPUPUPX</a>,</p>
<p>In order to check on your application, which we cannot do here, please reach out to our Education team via our contact form. Keep in mind, processing times may vary for applications, so thank you for your patience.</p>
